### ISSUE TITLE
Add a Likert scale component

### ISSUE THREAD

**Issue by csfar** [MyFormworks/Formworks#119]
It is common to have Likert-like scale in forms, Formworks should have such scales to developers. If added, please take into consideration accessibility. For more information, check [here](https://en.wikipedia.org/wiki/Likert_scale).
